packages/material-ui-lab/src/TreeItem/TreeItem.js

```diff
@@ -32,11 +32,27 @@ export const styles = theme => ({
         backgroundColor: 'transparent',
       },
     },
+    '&$disabled': {
```

Comment:
Something is off. I think that we should look into how we can rework the CSS/DOM structure to make the styles significantly simpler

Reply:
I agree. I couldn't figure out how to access `disabled` class inside `content` style object or `label` style, e.g
 ```js
content: {
    width: '100%',
    display: 'flex',
    alignItems: 'center',
    cursor: 'pointer',

    '&$disabled': {
      cursor: 'default',
    },
  },
 ```

The `disabled` style is not being added.